cli/launchpad/root.go

```diff
@@ -0,0 +1,85 @@
+package launchpad
+
+import (
+	"errors"
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"log"
+)
+
+//go:generate go run static/includestatic.go
+
+// crdKind is the CustomResourceDefinition (CRD) which is indicated YAML's Kind value.
+type crdKind string
+
+// outputFlavor defines launchpad's generated output language.
+type outputFlavor string
```

Comment:
This looks like it's being used as a enum. You should make it an enum then.

```go
type outputFlavor int
const (
   outDM outputFlavor = iota
   outTF
)
```

Same for crdKind.